### ISSUE TITLE
Cannot create archive with organization name

### ISSUE THREAD

**Issue by h2non** [h2non/nar#140]
Seems like using @orgname/name as package name generates a path error while extracting. Extract path must be normalized properly in those cases.

**Comment by SimenB** [comment]
@h2non How soon is soon? I don't, so I won't be able to provide a PR

**Comment by SimenB** [comment]
Adding ` console.log(path)` above where it fails gives the following stack trace:

```
/tmp/nar-pkg-1476410162047/@finn-no/cdn-uploader.tar
Error: The given path is not a file
    at /.nar/nar/lib/extract.js:62:9
    at /.nar/nar/node_modules/fw/lib/series.js:32:38
    at next (/.nar/nar/node_modules/fw/lib/series.js:33:15)
    at /.nar/nar/node_modules/fw/lib/util.js:40:35
    at /.nar/nar/node_modules/fw/lib/util.js:41:19
    at EventEmitter.extractEnd (/.nar/nar/lib/extract.js:122:16)
    at emitNone (events.js:86:13)
    at EventEmitter.emit (events.js:185:7)
    at Extract.onEnd (/.nar/nar/lib/unpack.js:17:22)
    at emitNone (events.js:91:20)
    at Extract.emit (events.js:185:7)
```

**Comment by h2non** [comment]
Ouch! Not before the weekend, I'm afraid.

**Comment by h2non** [comment]
Hey, will support this soon, I was a bit busy during the weekend.

**Comment by h2non** [comment]
Sorry for the delay. This should be fixed in `nar@0.3.38`.

Update `nar` running: `npm upgrade -g nar`.

**Comment by SimenB** [comment]
It works, thanks!
